packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -343,6 +344,30 @@ export function findSelectTokenButton(
     .should('have.text', text)
 }
 
+export function openTransactionDetails({
+  amount,
+  symbol
+}: {
+  amount: number
+  symbol: string
+}): Cypress.Chainable<JQuery<HTMLElement>> {
+  cy.findTransactionInTransactionHistory({ amount, symbol }).within(() => {
+    cy.findByLabelText('Transaction details button').click()
+  })
+  return cy.findByText('Transaction details').should('be.visible')
+}
+
+export function checkCustomAddress(
```

Comment:
this util is very specific to transaction details and can only be used there, so we should name it accordingly
```suggestion
export function findTransactionDetailsCustomDestinationAddress(
```